boards/x86/minnowboard/minnowboard.yaml

```diff
@@ -4,3 +4,4 @@ type: mcu
 arch: x86
 toolchain:
   - zephyr
+ram: 256
```

Comment:
why no flash size?

Reply:
we never flash anything on this board. This is loaded via EFI and can be in GBs